run.py

```diff
@@ -1,11 +1,20 @@
 from src.data_ingestion import prepare_data
 from src.plot_map import all_stations_map
 from src.download_data import download_data
+from src.first_analysis import analysis_1
+from src.second_analysis import analysis_2
 
 def main():
+    #Downloading data
     download_data("https://danepubliczne.imgw.pl/data/dane_pomiarowo_obserwacyjne/dane_meteorologiczne/dobowe/opad/")
+    #Data ingestion
     prepare_data()
+    #Map of all station locations
     all_stations_map()
+    #First analysis of data for chosen region
+    analysis_1("dolnośląskie")
```

Comment:
I really like the idea, but there is no need to display the charts - just log some information (e.g. how many stations is there) and then save the chart in e.g. visualization folder within repo (but the files inside shouldn't be added to the remote repo)